src/Mail/DriverInterface.php

```diff
@@ -44,14 +46,12 @@ public function availableSettings(): array;
      * presence of validation problems with the configured mail driver, Flarum
      * will fall back to its no-op {@see \Flarum\Mail\NullDriver}.
      */
-    // TODO: Uncomment for beta.13
-    //public function validate(SettingsRepositoryInterface $settings, Factory $validator): MessageBag;
+    public function validate(SettingsRepositoryInterface $settings, Factory $validator): MessageBag;
```

Comment:
Oh, sorry, I hadn't seen this PR until now. I already took care of this part, so please rebase on the latest `master`.